js/zb.js

```diff
@@ -456,6 +457,35 @@ module.exports = class zb extends Exchange {
                     },
                 },
             },
+            'options': {
+                'timeframes': {
+                    'spot': {
```

Comment:
i think we can just put this inside of the regular timeframes

Reply:
> i think we can just put this inside of the regular timeframes

No, because that will break the tests ad the users also rely on the documented structure of `timeframes` to match our Manual: https://github.com/ccxt/ccxt/wiki/Manual#ohlcv-candlestick-charts

@Dan-krm if possible, please, revert the last commit, let's keep them in `timeframes` (unified-to-unified mapping) and in `options` (unified-to-exchange-specific mappings per market type).